### PR TITLE
adding react-router-dom and react-bootstrap.

### PR DESCRIPTION
**Pull request by Doringl**
Fixes #27 

Changes proposed in this pull request : 

- adding react-router-dom 
- adding react-bootstrap 
- adding Wrapper component

@ayan-biswas0412/gssoc2021-HotelOnTouch

### CODE REVIEW

File: hotel-admin-frontend/src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Wrapper from '../components/Wrapper';
```
Comment (ayan-biswas0412) [inline hotel-admin-frontend/src/pages/HomePage.jsx:2]: import Wrapper from '../components'; // make a indx.js file in the ```components``` and export the components from there.

File: hotel-admin-frontend/src/App.js [hunk 2]
```diff
@@ -1,25 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
```
Comment (ayan-biswas0412) [inline hotel-admin-frontend/src/App.js:2]: import HomePage from './pages' // also make index.js inside pages and export all page from there

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
please change the file I have commented

**Review by ayan-biswas0412 — COMMENTED** [review]
your PR description is empty please follow the PR template

**Comment by Doringl** [comment]
I write a description.

**Comment by ayan-biswas0412** [comment]
> I write a description.

you haven't completed the onboarding process please do that, See the Contributing Section in Readme

**Comment by Doringl** [comment]
I cant write info about myself now im busy a bit, you merge if you want.

**Review by ayan-biswas0412 — APPROVED** [review]
nice

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
all your files are .jsx but it should be .js

**Comment by Doringl** [comment]
because they are return react component but I will fix it.

**Review by ayan-biswas0412 — APPROVED** [review]

**Pull request merged**